### PR TITLE
Documentation improvements in several files (D/DAR) 

### PR DESCRIPTION
**Pull request by cidrblock**
Part of an ongoing effort to improve documentation for future contributors:

doc strings updated to reflect parameters in function signature
descriptions updated to reflect functionality
fix spelling mistakes as encountered

This additionally allows for the removal of D* and DAR* exceptions from the flake8 configuration file for these files.

### CODE REVIEW

File: share/ansible_navigator/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Utility scripts that will be mounted into the execution environment.
```
Comment (webknjaz) [inline share/ansible_navigator/utils/__init__.py:1]: I don't think we need future possibilities in the title. Just tell people what it does not what something external does with it in some cases. At least, not in the title.
```suggestion
"""Utility scripts mounted into the execution environment.
```

File: share/ansible_navigator/utils/catalog_collections.py [hunk 2]
```diff
@@ -91,7 +93,12 @@ def _catalog_plugins(self, collection: Dict) -> None:
 
     @staticmethod
     def _generate_chksum(file_path: str, relative_path: str) -> Dict:
-        """generate a std checksum for a file"""
+        """Generate a standard checksum for a file.
+
+        :param file_path: The path to the file to generate a checksum for
+        :param relative_path: The relative path within the collection directory structure
+        :returns: A diction or details about the file, including the checksum
```
Comment (webknjaz) [inline share/ansible_navigator/utils/catalog_collections.py:100]: "diction or details"? Feels like some autocorrect gone rogue. "Of"?

**Review by webknjaz — COMMENTED** [review]
I did not look closely at everything but some of the patterns I've pointed out repeat across the patch.

File: share/ansible_navigator/utils/catalog_collections.py (hunk 2 above)
Comment (cidrblock) [reply share/ansible_navigator/utils/catalog_collections.py:100]: lol

**Review by webknjaz — APPROVED** [review]

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**